src/cca/islet_test/attest_seal_test.cc

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the language governing permissions and
+ *  limitations under the License
+ */
+
+#include <islet.h>
+
+#include <iostream>
+#include <string>
+#include <cstring>
+
+#define BUFFER_SIZE 2048
```

Comment:
Please, can you document where we arrived at this limit from? The reason for these kinds of sizes will be lost over time, so even if you can add a simple comment, something like:

// Some reasonable size to allocate <... fill-in-the-blank> ... on-stack buffers.

If you can give the reason why 2K is big enough, that will be helpful. (For future readers.)

Reply:
I added it. Thank you for the suggestion!